src/fastoad/openmdao/validity_checker.py

```diff
@@ -169,18 +171,23 @@ def check_problem_variables(cls, problem: om.Problem) -> List[CheckRecord]:
         return records
 
     @classmethod
-    def check_variables(cls, variables: VariableList) -> List[CheckRecord]:
+    def check_variables(
+        cls, variables: VariableList, activated_only: bool = True
+    ) -> List[CheckRecord]:
         """
         Check values of provided variables against registered limits.
 
         :param variables:
+        :param ignore_deactivated:
```

Comment:
The name of the option in the comment does not match the signature of the method. Additionally, wht does this parameter does ?

Reply:
doscstring fixed in 1fbf6a528fbb9bbb0b216c36139cfa0023be121f